subfun/BIDS_rmDummies.m

```diff
@@ -36,14 +36,15 @@ function BIDS_rmDummies(opt)
 
     % prefix to append to the files with dummy removed
     prefix = getPrefix('rmDummies',opt);
-    
+
     numDummies = opt.numDummies; % Number of dummies
 
     %% Loop through the groups, subjects, sessions
     for iGroup= 1:length(group)             % For each group
         groupName = group(iGroup).name ;    % Get the group name
 
         for iSub = 1:group(iGroup).numSub   % For each Subject in the group
+            cd(WD);   % Go to scripts directory to run getSessions & GetRuns functions
```

Comment:
The script goes to the directory of the functional files. 
So I needed to return to the scripts WD to run the getRuns/GetSessions

Reply:
Thats my fault., I commented the part of  "adding them to the path" because it was your folder, and didnt replace it with mine. Its my fault.
Yes, I will add it to my path, and then all the jumping across directories wont be necessary.